Gruntfile.js

```diff
@@ -69,6 +70,18 @@ module.exports = exports = function(grunt) {
                         // just to add the BANNER
                         // without adding an extra grunt module
                         'dist/localforage.nopromises.js'
+                    ],
+                    // Node output files
+                    'dist/localforage.node.js': [
+                        'src/utils/node_banner.js',
+                        'bower_components/es6-promise/promise.js',
+                        'dist/localforage.nopromises.js'
+                    ],
+                    'dist/localforage.nopromises.node.js': [
+                        // just to add the BANNER
+                        // without adding an extra grunt module
+                        'src/utils/node_banner.js',
+                        'dist/localforage.nopromises.js'
```

Comment:
Actually it just occurred to me that we can solve this problem by simply changing the `"main"` in the `package.json` to point to the Node distributable version.